### ISSUE TITLE
TeamsChannelData object does not contain definition for OnBehalfOf property

### ISSUE THREAD

**Issue by Prithvi-MSFT** [MicrosoftDocs/msteams-docs#6414]
TeamsChannelData class in C# does not contain definition for OnBehalfOf property.
Below is sample json.

```
"channelData": {
           onBehalfOf: [
             { 
               itemId: 0, 
               mentionType: 'person', 
               mri: context.activity.from.id, 
               displayname: context.activity.from.name 
             `}`
            ],
           "notification": {
           "alertInMeeting": true,
           "externalResourceUrl": "https://teams.microsoft.com/l/bubble/APP_ID?url=<url>&height=<height>&width=<width>&title=<title>&completionBotId=BOT_APP_ID"
            }
        }
```

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 62ce7ca6-3ea8-ccca-5c59-92a638303aea
* Version Independent ID: d9a2d747-41cc-add6-f281-4210e888654e
* Content: [Enable and configure your apps for Teams meetings - Teams](https://review.docs.microsoft.com/en-us/microsoftteams/platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings?branch=pr-en-us-5894#in-meeting-notification)
* Content Source: [msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/main/msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md)
* Service: **msteams**
* GitHub Login: @surbhigupta
* Microsoft Alias: **lajanuar**

**Comment by Prasad-MSFT** [comment]
@Prithvi-MSFT - We have raised a bug for the same and engineering team is looking into it. We will inform you once we get any further update. Thanks!

**Comment by Prasad-MSFT** [comment]
Hi @Prithvi-MSFT - We got update from engineering team that; this is fixed in 4.20.0 SDK. Please do verify.
[Release Bot Framework DotNet SDK 4.20.0 · microsoft/botbuilder-dotnet (github.com)](https://github.com/microsoft/botbuilder-dotnet/releases/tag/4.20.0)
[TeamsChannelData need OnBehalfOf · Issue #6596 · microsoft/botbuilder-dotnet (github.com)](https://github.com/microsoft/botbuilder-dotnet/issues/6596)
